src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -1,30 +1,76 @@
 import { Registration, RegistrationStatus } from '../../types/registration.types';
-import { Box, Typography } from '@mui/material';
+import { Box, Link, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
+import { getTitleString } from '../../utils/registration-helpers';
+import { useQuery } from '@tanstack/react-query';
+import { fetchRegistration } from '../../api/registrationApi';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { PageSpinner } from '../../components/PageSpinner';
 
 interface DeletePublicationInformationProps {
   registration: Registration;
 }
 
+function extractOriginalIdentifier(registration: Registration): string | null {
+  return registration.duplicateOf ? getIdentifierFromId(registration.duplicateOf) : null;
+}
+
 export const DeletedPublicationInformation = ({ registration }: DeletePublicationInformationProps) => {
   const { t } = useTranslation();
+
+  const originalIdentifier = extractOriginalIdentifier(registration);
+
+  const originalRegistrationQuery = useQuery({
+    queryKey: ['registration', originalIdentifier],
```

Comment:
Må være rart å komme fra backend til frontend og plutselig switche om fra å kalle `publication` for `registration`? 😅

Reply:
Jepp. Registration er noe man gjør, publication er en ting i midt hode.